aten/src/ATen/DynamicLibrary.cpp

```diff
@@ -48,10 +49,11 @@ DynamicLibrary::DynamicLibrary(const char* name) {
   // NOLINTNEXTLINE(hicpp-signed-bitwise)
   HMODULE theModule;
   bool reload = true;
+  auto wname = c10::u8u16(name);
   // Check if LOAD_LIBRARY_SEARCH_DEFAULT_DIRS is supported
-  if (GetProcAddress(GetModuleHandle("KERNEL32.DLL"), "AddDllDirectory") != NULL) {
-    theModule = LoadLibraryExA(
-        name,
+  if (GetProcAddress(GetModuleHandleW(L"KERNEL32.DLL"), "AddDllDirectory") != NULL) {
```

Comment:
nit: This change is not necessary, right?

Reply:
Yeah I just want to make sure other people would choose `GetModuleHandleW` in similar places in the future.